[2023-12-16T05:40:20.403+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: assignment_spark_dag.run_spark_task manual__2023-12-16T05:39:48.300145+00:00 [queued]>
[2023-12-16T05:40:20.409+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: assignment_spark_dag.run_spark_task manual__2023-12-16T05:39:48.300145+00:00 [queued]>
[2023-12-16T05:40:20.410+0000] {taskinstance.py:1359} INFO - Starting attempt 2 of 2
[2023-12-16T05:40:20.423+0000] {taskinstance.py:1380} INFO - Executing <Task(BashOperator): run_spark_task> on 2023-12-16 05:39:48.300145+00:00
[2023-12-16T05:40:20.427+0000] {standard_task_runner.py:57} INFO - Started process 1566 to run task
[2023-12-16T05:40:20.429+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'assignment_spark_dag', 'run_spark_task', 'manual__2023-12-16T05:39:48.300145+00:00', '--job-id', '70', '--raw', '--subdir', 'DAGS_FOLDER/assignment_spark_dag.py', '--cfg-path', '/tmp/tmp2gbj___1']
[2023-12-16T05:40:20.431+0000] {standard_task_runner.py:85} INFO - Job 70: Subtask run_spark_task
[2023-12-16T05:40:20.442+0000] {logging_mixin.py:151} WARNING - /home/airflow/.local/lib/python3.9/site-packages/airflow/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-12-16T05:40:20.466+0000] {task_command.py:415} INFO - Running <TaskInstance: assignment_spark_dag.run_spark_task manual__2023-12-16T05:39:48.300145+00:00 [running]> on host dataeng-airflow-scheduler
[2023-12-16T05:40:20.529+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ahmad_belva' AIRFLOW_CTX_DAG_ID='assignment_spark_dag' AIRFLOW_CTX_TASK_ID='run_spark_task' AIRFLOW_CTX_EXECUTION_DATE='2023-12-16T05:39:48.300145+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-12-16T05:39:48.300145+00:00'
[2023-12-16T05:40:20.530+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-12-16T05:40:20.531+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'make spark-submit-test']
[2023-12-16T05:40:20.538+0000] {subprocess.py:86} INFO - Output:
[2023-12-16T05:40:20.539+0000] {subprocess.py:93} INFO - /bin/bash: line 1: make: command not found
[2023-12-16T05:40:20.539+0000] {subprocess.py:97} INFO - Command exited with return code 127
[2023-12-16T05:40:20.548+0000] {taskinstance.py:1935} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/operators/bash.py", line 210, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 127.
[2023-12-16T05:40:20.550+0000] {taskinstance.py:1398} INFO - Marking task as FAILED. dag_id=assignment_spark_dag, task_id=run_spark_task, execution_date=20231216T053948, start_date=20231216T054020, end_date=20231216T054020
[2023-12-16T05:40:20.562+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 70 for task run_spark_task (Bash command failed. The command returned a non-zero exit code 127.; 1566)
[2023-12-16T05:40:20.602+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-12-16T05:40:20.614+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
